pkg/config/config.go

```diff
@@ -300,6 +300,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("dogstatsd_string_interner_size", 4096)
 	// Enable check for Entity-ID presence when enriching Dogstatsd metrics with tags
 	config.BindEnvAndSetDefault("dogstatsd_entity_id_precedence", false)
+	config.BindEnvAndSetDefault("dogstatsd_disable_verbose_logs", false)
```

Comment:
What do you think about adding something to the config file to say it apply only on parsing error?

Reply:
I'll add a comment to this line mentioning this 👍